wikipedia-dark.user.css

```diff
@@ -447,7 +450,8 @@ domain("wiki.rpcs3.net") {
             filter: invert(100%) !important;
     background-color: transparent !important;
   }
-  .central-textlogo__image, .unicode.haudio a img {
+  .central-textlogo__image, .unicode.haudio a img,
+  .mw-ui-icon-popups-close::before {
```

Comment:
Single colon as we do with other similar entries. =)

Reply:
Wasn't sure since it seemed like a mix of both one and two colon selectors were already in use for pseudo-elements. ``::selection`` and ``::-moz-selection`` are using two.

Single-colon syntax for pseudo-elements is also [deprecated in CSS3](https://www.w3.org/TR/selectors/#pseudo-element-syntax).